contracts/contracts/harvest/SuperOETHHarvester.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import { OETHHarvesterSimple, IERC20, IStrategy, SafeERC20 } from "./OETHHarvesterSimple.sol";
+
+contract SuperOETHHarvester is OETHHarvesterSimple {
+    using SafeERC20 for IERC20;
+
+    constructor(address _wrappedNativeToken)
```

Comment:
Since it doesn't do anything extra is this constructor required?

Reply:
I think, yes, since it specifies a parameter to be set. I get a `TypeError` if I don't have it:
```
TypeError: No arguments passed to the base constructor. Specify the arguments or mark "SuperOETHHarvester" as abstract.
 --> contracts/harvest/SuperOETHHarvester.sol:6:1:
  |
6 | contract SuperOETHHarvester is OETHHarvesterSimple {
  | ^ (Relevant source part starts here and spans across multiple lines).
Note: Base constructor parameters:
  --> contracts/harvest/OETHHarvesterSimple.sol:49:16:
   |
49 |     constructor(address _wrappedNativeToken) {
   |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

```

Reply:
Ah interesting thanks for testing it